action.yml

```
@@ -36,6 +36,10 @@ inputs:
     description: 'The action will use distgen for generating dockerfiles if true'
     required: false
     default: 'false'
+  suffix:
+    description: 'The action will add suffix to image name'
```

```suggestion
    description: 'A suffix to the image name. Also used for the excluding mechanism, see README.md'
```
I can adjust the README separately.